composer.json

```diff
@@ -15,21 +15,25 @@
     },
     "pluswerk/grumphp-config": {
       "auto-setting": true
+    },
+    "branch-alias":{
+      "dev-v2": "2.0.x-dev"
     }
   },
   "config": {
     "sort-packages": true
   },
   "autoload": {
     "psr-4": {
-      "Pluswerk\\Sentry\\": "Classes/"
+      "Plus\\PlusSentry\\": "Classes/"
```

Comment:
why not Pluswerk?
https://github.com/pluswerk/mail_logger/blob/master/composer.json#L44
https://github.com/pluswerk/cache_automation/blob/master/composer.json#L18
https://github.com/pluswerk/minify/blob/master/composer.json#L12